compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -490,7 +490,7 @@ OMR::Z::CodeGenerator::CodeGenerator()
       }
    self()->setSupportsArrayCmp();
    self()->setSupportsArrayCmpSign();
-   if (!comp->compileRelocatableCode())
+   if (!(comp->compileRelocatableCode() || comp->isOutOfProcessCompilation()))
```

Comment:
This `isOutOfProcessCompilation` only seems to exist in OpenJ9 as far as I can tell. See Travis build failure. Do we need to overload in OMR?

Reply:
I think we'll need a different query here (I'm not sure what to use yet). I should have disabled `ci` tests from running on this and opened this as a draft PR.